nubis/bin/example-consul2unixusers.conf

```diff
@@ -0,0 +1,5 @@
+[consul]
+server = hostname.for.your.consul.server
+scheme = http
+port = 80
```

Comment:
The config file variables are formatted to make sense for the way python-consul wants the values. You can't pass a http://localhost:8500/ style url to python-consul (or if you can, that's not the usual way of interacting with it.) You have to specify host, port, and transport separately. It didn't seem worth my time, on this timeline, to write code to separate out the URL from the config file.